pkg/okteto/secrets.go

```diff
@@ -320,3 +327,58 @@ func (c *userClient) GetExecutionEnv(ctx context.Context) (map[string]string, er
 	}
 	return result, nil
 }
+
+func getRegistryAndRepositoryFromImage(image string) (string, error) {
```

Comment:
can we add a comment describing what kind on image strings are valid input, and what we can expect for the function to extract? as an example? 

i am not sure this should be the place for this funciton. there are places we are doing this same operation? perhaps a pkg where we only export this and can use it as common util for image string?

Reply:
Yes, I think it would be good to have it under the registry/image functionality but there is a import cycle that right now there is no way of untangle it